Fireworks/Core/src/FWGUIManager.cc

```diff
@@ -642,6 +645,36 @@ FWGUIManager::showEDIFrame(int iToShow)
    m_ediFrame->MapRaised();
 }
 
+
+void
+FWGUIManager::open3DRegion()
+{
+   FWModelId id =  *(m_context->selectionManager()->selected().begin());
+   float eta =0, phi = 0;
+   {
+      edm::TypeWithDict type = edm::TypeWithDict((TClass*)id.item()->modelType());
+      using namespace boost::spirit::classic;
+      reco::parser::ExpressionPtr tmpPtr;
+      reco::parser::Grammar grammar(tmpPtr,type);
+      edm::ObjectWithDict o(type, (void*)id.item()->modelData(id.index()));
+      try {
+         parse("theta()", grammar.use_parser<1>() >> end_p, space_p).full;
```

Comment:
Does it really work?
According to the Clang 664 and 666 line expressions are not used:
```
Fireworks/Core/src/FWGUIManager.cc:664:70: error: expression result unused [-Werror,-Wunused-value]
Fireworks/Core/src/FWGUIManager.cc:666:68: error: expression result unused [-Werror,-Wunused-value]
```
This is crashing CLANG IBs.

I guess you need if-statement around `parse()` to at least calm down the Clang, e.g.
```
111    if(m_type != edm::TypeWithDict() && iExpression.size())
112    {
113       using namespace fireworks::expression;
114
115       //Backwards compatibility with old format
116       std::string temp = oldToNewFormat(iExpression);
117
118       //now setup the parser
119       try
120       {
121          if(parse(temp.c_str(), grammar.use_parser<1>() >> end_p, space_p).full)
122          {
123             m_entries.push_back(Entry(tmpPtr, iExpression, iUnit, iTitle.empty() ? iExpression :iTitle , iPrec));
124             m_titleWidth = TMath::Max(m_titleWidth, (int) m_entries.back().m_title.size());
125             return true;
126          }
127       }
128       catch(const reco::parser::BaseException& e)
129       {
130          // std::cout <<"failed to parse "<<iExpression<<" because "<<reco::parser::baseExceptionWhat(e)<<std::endl;
131       }
132    }
133    return false;
```